pyproject.toml

```diff
@@ -8,14 +8,14 @@ requires = [
 name = "python-docs-theme"
 description = "The Sphinx theme for the CPython docs and related projects"
 readme = "README.md"
-license.file = "LICENSE"
+license = "PSF-2.0"
+license-files = [ "LICENSE" ]
```

Comment:
I don't think we need to set this, flit has a sensible default. Doesn't hurt, though.